sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/AadWebSecurityConfigurerAdapter.java

```diff
@@ -114,9 +122,14 @@ protected LogoutSuccessHandler oidcLogoutSuccessHandler() {
     protected OAuth2AccessTokenResponseClient<OAuth2AuthorizationCodeGrantRequest> accessTokenResponseClient() {
         DefaultAuthorizationCodeTokenResponseClient result = new DefaultAuthorizationCodeTokenResponseClient();
         if (repo instanceof AadClientRegistrationRepository) {
-            result.setRequestEntityConverter(
+            AadOAuth2AuthorizationCodeGrantRequestEntityConverter converter =
```

Comment:
Can we use a more OO way in this place?

Reply:
Please provide your proposal for reference.